webapp/channels/src/components/widgets/menu/menu_items/restricted_indicator.tsx

```diff
@@ -55,6 +61,7 @@ const RestrictedIndicator = ({
     customSecondaryButtonInModal,
     feature,
     minimumPlanRequiredForFeature,
+    padding = '0 10px',
```

Comment:
Could this be done with CSS automatically based on the parent class?

Reply:
agreed, i verified again this is actually not needed